covid_xprize/validation/predictor_validation.py

```diff
@@ -68,12 +69,18 @@ def _check_columns(expected_columns, pred_df):
     missing_columns = expected_columns - set(pred_df.columns)
     if missing_columns:
         errors.append(f"Missing columns: {missing_columns}")
+        # Columns are not there, can't check anything more
         return errors
+
+    # Make sure column Date contains dates
+    date_column_type = pred_df[DATE].dtype
+    if not np.issubdtype(date_column_type, np.datetime64):
+        errors.append(f"Column {DATE} contains non date values: {date_column_type}")
```

Comment:
Would be helpful (but a bit fiddly) to display a couple of examples of the errant values here in the error message.

Reply:
Could be an improvement. If we need it.